robottelo/constants.py

```diff
@@ -1187,3 +1187,31 @@
     u'pulp_data.tar',
     u'pulp.snar',
 ]
+
+TOOLS_ERRATA_DETAILS = [
+    ['Advisory', 'RHBA-2016:1503'],
+    ['CVEs', 'N/A'],
+    ['Type', 'Bug Fix Advisory'],
+    ['Severity', 'None'],
+    ['Issued', '7/27/16'],
+    ['Last Updated On', '7/27/16'],
+    ['Reboot Suggested', 'No'],
+    [
+        'Topic',
+        'Red Hat Satellite 6.2 now available for Red Hat Enterprise Linux 6 '
+        'and 7'
+    ],
+    ['Description', 'This update provides Satellite 6.2 client tooling'],
+    [
+        'Solution',
+        'Before applying this update, make sure all previously released errata'
+    ],
+]
+
+TOOLS_ERRATA_TABLE_DETAILS = [
+    'RHBA-2016:1503',
+    'Satellite 6.2 Tools Release',
+    'Bug Fix Advisory',
+    '0 Applicable, 0 Installable',
```

Comment:
Not sure about this line. Some test running in the background can create a host which will be applicable for this errata (have corresponding package installed). Can see multiple solutions here:
a) if package affected by this errata is not used in any test then no issues here
b) if package is used in other tests we can try find another one which will be unique
c) if no unique package available - we should probably think of validating this field by some mask ('# Applicable, # Installable')